src/components/button/utils.tsx

```diff
@@ -6,8 +6,10 @@ import { LoadingIcon } from '../loading';
 export type ButtonModes =
   | 'interactive_0'
   | 'interactive_0--bordered'
+  | 'interactive_0--bordered_alternative'
   | 'interactive_1'
   | 'interactive_2'
+  | 'interactive_2_alternative'
```

Comment:
Variations should be named by separating the base and variant with double hyphens; for example 'interactive_2--alternative'`.

Furthermore, I think "alternative" might be a bit vague and not descriptive enough, but at the same time I don't have any good proposed solution so it's not a big deal.

Reply:
Okey, I will change the naming, and keep that in minde when naming .css classes in the future. 

I'll also see if I can find a better name. Initially, I thought of a name to distinguish between light and dark background, but it didn't really fit with its color palette.